mongoengine/fields.py

```diff
@@ -1453,8 +1512,11 @@ class GenericReferenceField(BaseField):
         * You can use the choices param to limit the acceptable Document types
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, reverse_delete_rule=DO_NOTHING, **kwargs):
```

Comment:
Would like to call this param out in the `GenericReferenceField` docs but I couldn't seem to get it to show up properly, I'd make a docstring for the init but it wouldn't populate to the docs 🤷